web/server/codechecker_server/api/report_server.py

```diff
@@ -2101,6 +2101,33 @@ def removeRun(self, run_id):
                      self.__get_username())
         return True
 
+    @exc_to_thrift_reqfail
+    @timeit
+    def updateRunData(self, run_id, new_run_name):
```

Comment:
If `new_run_name` refers an existing run, CodeChecker throws an exception from the database.